src/map/modifier.h

```diff
@@ -836,7 +836,7 @@ enum class Mod
     ITEM_ADDEFFECT_CHANCE   = 501, // Chance of an items Additional Effect or Spikes
     ITEM_ADDEFFECT_ELEMENT  = 950, // Element of the Additional Effect or Spikes, for resist purposes
     ITEM_ADDEFFECT_STATUS   = 951, // Status Effect ID to try to apply via Additional Effect or Spikes
-    ITEM_ADDEFFECT_POWER    = 952, // Base Power for effect in MOD_ITEM_ADDEFFECT_STATUS
+    ITEM_ADDEFFECT_POWER    = 952, // Base Power for effect in MOD_ITEM_ADDEFFECT_STATUS. Must be used for debuffs/buffs.
```

Comment:
I had thought it obvious because the comment outright stating its for the status, but if that helps you remember to use it, ok.

Reply:
Just trying to make it more clear for others that contribute, why I specified debuffs/buffs. From our discussion, it seems this is a "Do damage, not needed, do debuff, needed" issue. Well, without all those words.